code/nodes/preprocParams/preprocParams.q

```diff
@@ -2,11 +2,11 @@
 // such they can be consolidated into a single node later in the workflow
 \d .automl
 
-inputKeys :`config`dataDescription`creationTime`sigFeats`symEncode`symMap
-inputTypes:"!+tSS!"
+inputKeys :`config`dataDescription`creationTime`sigFeats`symEncode`symMap`featModel
+inputTypes:"!+tSS!<"
 preprocParams.node.inputs  :inputKeys!inputTypes
 preprocParams.node.outputs :"!"
-preprocParams.node.function:{[cfg;descrip;ctime;sigfeat;symenc;symmap]
+preprocParams.node.function:{[cfg;descrip;ctime;sigfeat;symenc;symmap;featmodel]
```

Comment:
Function needs to have the 'pre-function description' added styled similar to the featureDescription.q node

Reply:
This node isn't completed yet, and "pre-function description" is usually added by owner of node (Deanna in this case)

Reply:
Noted, didn't take into account it was a different node than the featureCreation